target_goal:
  eleven_two: {pose: [44.129733974, 5.25948360461, 0.0], orientation: [0.0, 0.0, -0.0532056699849, 0.998583575211]}
  eleven_one: {pose: [40.7150111824, 14.0503073292, 0.0], orientation: [0.0, 0.0, 0.686784206731, 0.726861371504]}
  west_hall: {pose: [28.6653486518, 12.9132832033, 0.0], orientation: [0.0, 0.0, 0.750366573973, 0.661021939624]}
  ten_one: {pose: [8.20520890439, 7.45247223399, 0.0], orientation: [0.0, 0.0, -0.99663499056, 0.0819676496615]}
  liabray_1: {pose: [26.9620667232, -6.45611130925, 0.0], orientation: [0.0, 0.0, 0.0048406774769, 0.999988283852]}
  eight_six: {pose: [49.9133742559, -12.5244231699, 0.0], orientation: [0.0, 0.0, -0.0741591862521, 0.997246416436]}
  liabray_2: {pose: [37.7311677239, -3.5762595658, 0.0], orientation: [0.0, 0.0, -0.999996772544, 0.00254064978956]}
  liabray_3: {pose: [30.2298295235, -6.99752018758, 0.0], orientation: [0.0, 0.0, -0.999154940338, 0.0411023746016]}
  ten_two: {pose: [7.63090454323, -11.6423009978, 0.0], orientation: [0.0, 0.0, -0.751537486896, 0.65969038631]}
  six_two: {pose: [12.4877273133, -33.8648884818, 0.0], orientation: [0.0, 0.0, -0.664530486699, 0.74726115398]}
  world-cup: {pose: [25.7502471181, -13.5170212956, 0.0], orientation: [0.0, 0.0, 0.622151118284, 0.782897174614]}
  eight_five: {pose: [49.2445623935, -26.2171985001, 0.0], orientation: [0.0, 0.0, 0.0349430676916, 0.999389304536]}
  three_one: {pose: [43.2747301499, -42.8061095766, 0.0], orientation: [0.0, 0.0, -0.994845703157, 0.101400329932]}
  community: {pose: [12.6145432925, -45.6549623155, 0.0], orientation: [0.0, 0.0, -0.804125656992, 0.594459357541]}
  community_2: {pose: [22.8609726846, -46.151037846, 0.0], orientation: [0.0, 0.0, -0.722792357035, 0.691065270877]}
  community_youth_meeting: {pose: [7.13664498692, -59.7164473245, 0.0], orientation: [0.0, 0.0, 0.999999227246, 0.00124318439656]}
  two_one: {pose: [11.4634750845, -68.1062137641, 0.0], orientation: [0.0, 0.0, -0.71611073101, 0.697986691086]}
  east_hall: {pose: [23.4234284077, -66.4768116213, 0.0], orientation: [0.0, 0.0, -0.451963411575, 0.892036476046]}
  three_four: {pose: [41.6825672076, -59.4114625808, 0.0], orientation: [0.0, 0.0, 0.0862983689059, 0.996269336838]}
  five_one: {pose: [31.1082947169, -42.8236707346, 0.0], orientation: [0.0, 0.0, 0.998560404012, 0.0536387876377]}
  one: {pose: [20.6771945953, -10.9838047028, 0.0], orientation: [0.0, 0.0, -0.0015329492726, 0.999981158123]}
  two: {pose: [19.4531726837, -25.8618164062, 0.0], orientation: [0.0, 0.0, 0.99990603116, 0.00899602107519]}
  three: {pose: [6.96866798401, -16.5190315247, 0.0], orientation: [0.0, 0.0, 0.694624927514, 0.719372094313]}
  four: {pose: [31.1082947169, -42.8236707346, 0.0], orientation: [0.0, 0.0, 0.998560404012, 0.0536387876377]}
  five: {pose: [31.1082947169, -42.8236707346, 0.0], orientation: [0.0, 0.0, 0.998560404012, 0.0536387876377]}